lib/tether/tether.go

```diff
@@ -251,8 +251,18 @@ func (t *tether) setHostname() error {
 }
 
 func (t *tether) setNetworks() error {
+	var networks []*NetworkEndpoint
+	// prioritize internal networks
 	for _, v := range t.config.Networks {
```

Comment:
from the future proof stand point if user end-up using tens of thousands of networks it's better to go over local networks first and put them to the front and in the second loop add the rest, so you would avoid making bunch of large copies. The is a little more efficient way of doing this in one pass, but it will make this code a little less readable. However, you may accept the challenge.